spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -2368,14 +2368,25 @@ def setup_context_menu(self):
                     (self.copy_action, None, selectall_action,
                      self.gotodef_action))
 
+    def focusOutEvent(self, event):
+        """Override Qt method."""
+        self.scrollflagarea.update()

```

Comment:
Great, thanks!

Reply:
Ok, I've done it. Do you want me to do that to do that for other logics (e.g. refreshing the flags when something change in the editor) even if it is not directly related to this PR?

Reply:
Hmm no, only the ones you added, in the overrided events

Although if you want to move the others It will be great, the `CodeEditor` still need a lot of refactoring and moving logic out to panels or extensions.

Reply:
ok I'll do it, it is not a big change to make.